manifests/a/AgileBits/1PasswordCLI/2.12.0/AgileBits.1PasswordCLI.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: AgileBits.1PasswordCLI
```

```suggestion
PackageIdentifier: AgileBits.1Password.CLI
```
This should probably be a subpackage of AgileBits.1Password